scripts/amd/gemm/tune_gemm.py

```diff
@@ -336,10 +375,12 @@ def main():
         prog="tune a specific gemm size",
         allow_abbrev=False,)
     parser.add_argument("-n", type=int, default=1, help='number of threads')
+    parser.add_argument("-rotating_tensor", type=int, default=256, help='size of rotating buffer (MB), default: 256')
```

Comment:
Since this is generated by the tuning script, I'd suggest we set the default value to be `rotating_buffer_size`. Therefore, when users want to debug by running `python generated_kernelM-N-K-0.py`, they don't need to provide the rotating buffer size.

Reply:
Thanks, fixed.